src/promptflow-devkit/promptflow/_cli/_completers/_param_completers.py

```diff
@@ -0,0 +1,21 @@
+from argcomplete import warn
+
+from promptflow._sdk._constants import ListViewType
+from promptflow.client import PFClient
+
+MAX_RESULTS = 20
+
+
+def run_name_completer(prefix, parsed_args, **kwargs):
+    client = PFClient()
+    runs = client.runs.list(list_view_type=ListViewType.ALL)
```

Comment:
do a max_results=MAX_RESULTS +1 here? So we won't query too much rows from the db in memory.

Reply:
Hello, I ended up by writing a new "search" method in the client that will search by name prefix.
I removed the "warn" for too many results since it breaks the view for some reason -> the method is now much simpler